### PR TITLE
Create a placeholder site.

### PR DESCRIPTION
**Pull request by mjhenkes**
### Summary
Add a dummy site for future pull requests to add content to.
Enable auto gh-pages deployment
Enable Heroku PR deployments

Thanks for contributing to Terra! -->

[CONTRIBUTORS.md]: https://github.com/cerner/terra-dev-site/blob/master/CONTRIBUTORS.md
[License]: https://github.com/cerner/terra-dev-site/blob/master/License.md

### CODE REVIEW

File: themeable-variables.json [hunk 1]
```diff
@@ -0,0 +1,207 @@
+{
```
Comment (emilyrohrbough) [inline themeable-variables.json]: Do we want to check this in with the dev-site?

Comment (mjhenkes) [reply themeable-variables.json]: probably not, but it annoyingly get generated.

File: package.json [hunk 2]
```diff
@@ -43,17 +43,23 @@
     "clean": "node scripts/clean-modules/index.js",
     "clean:install": "npm run clean && npm install",
     "clean:obsolete-snapshots": "npm test -- -u",
+    "compile:heroku": "webpack --config src/config/webpack.prod.config -p",
+    "compile:prod": "webpack --config src/config/webpack.prod.config -p",
     "danger": "danger",
+    "heroku-postbuild": "npm install --only=dev && npm run compile:heroku",
+    "heroku-prebuild": "npm install rimraf",
```
Comment (emilyrohrbough) [inline package.json]: Do we need this? I don't think we are compiling anywhere?

File: package.json [hunk 3]
```diff
@@ -43,17 +43,23 @@
     "clean": "node scripts/clean-modules/index.js",
     "clean:install": "npm run clean && npm install",
     "clean:obsolete-snapshots": "npm test -- -u",
+    "compile:heroku": "webpack --config src/config/webpack.prod.config -p",
+    "compile:prod": "webpack --config src/config/webpack.prod.config -p",
     "danger": "danger",
+    "heroku-postbuild": "npm install --only=dev && npm run compile:heroku",
```
Comment (emilyrohrbough) [inline package.json]: Could we just use `compile:prod` for `heroku-postbuild`? `compile:prod` and `compile:heroku` are the same script

File: package.json (hunk 2 above)
Comment (mjhenkes) [reply package.json]: Maybe not, but i was thinking we might want to delete the 'build' folder on clean eventually.

File: package.json (hunk 3 above)
Comment (mjhenkes) [reply package.json]: Seems reasonable. We could update this in the mono-repos too.

**Comment by emilyrohrbough** [comment]
I see it deployed fine without a Profile or app.json. Did you set that up in heroku?

File: themeable-variables.json (hunk 1 above)
Comment (emilyrohrbough) [reply themeable-variables.json]: yeah...

**Comment by mjhenkes** [comment]
I might have committed the app.json and procfile directly to master to make sure this worked.

**Pull request merged**